packages/flutter/test/material/scaffold_test.dart

```diff
@@ -1550,6 +1550,7 @@ void main() {
 
       scaffoldState.openDrawer();
       expect(true, scaffoldState.isDrawerOpen);
+      await tester.pump();
```

Comment:
Did this test fail if you didn't add this line?  I wonder if maybe it should go one line above, above the `expect`...

Reply:
Yes, this supplement prevents getting the element in the defunct state. I've done it.